start.sh

```diff
@@ -54,23 +48,15 @@ done
 echo Running python migrations
 python manage.py migrate
 
-echo "Setting domain of default site record"
 
 if [ "${DEBUGPY_ENABLE:-"false"}" == "false" ]; then
-    echo "Starting Gunicorn for production"
+    echo "Starting Gunicorn with uvicorn worker for production"
+    CMD="gunicorn backend.asgi:application --bind 0.0.0.0:${GUNICORN_PORT} --workers=\"${GUNICORN_WORKERS}\" -k uvicorn_worker.UvicornWorker --timeout=\"${GUNICORN_TIMEOUT}\" "
 else
-    echo "Starting Gunicorn for DEBUGPY debugging"
-    # Workers need to be set to 1 for DEBUGPY
-    GUNICORN_WORKERS=1
-    GUNICORN_RELOAD="--reload"
-    GUNICORN_TIMEOUT=0
+    echo "Starting uvicorn for Development"
+    CMD="uvicorn backend.asgi:application --host=0.0.0.0 --port=${GUNICORN_PORT} --reload"
```

Comment:
Here is the author recommendation https://github.com/Kludex/uvicorn-worker?tab=readme-ov-file#development